en/docs/guides/identity-federation/twitter.md

```diff
@@ -15,40 +15,165 @@ This guide assumes you have your own applications. If you wish to try out this f
+    <p>The acs indicates the Assertion Consumer URL of the WSO2 Identity Server endpoint that accepts the responses sent by Twitter.</p></td>
+    <td><code>                               https://wso2.com:9443/commonauth                             </code></td>
+    </tr>
+    </tbody>
+    </table>
+
+5.  Click **Register**.
 
 ---
 
 ## Configure the service provider
 
-(insert-admin-portal-link)
+1.  Access the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`).
+2.  On the **Main** menu, click **Identity \> Service Providers \>**
+    **Add**.
+3.  Fill in the **Service Provider Name** and provide a brief
+    **Description** of the service provider. Only **Service Provider
+    Name** is a required field and you can use Twitter-SP as the name for
+    this example.
+4.  Click **Register**.
+5.  Expand the **Inbound Authentication Configuration** and the **SAML2
+    Web SSO Configuration**, and click **Configure**.
+6.  In the form that appears, fill out the following configuration
+    details required for single sign-on. For more details about
+    attributes in the following configuration refer [SAML2 Web SSO Configuration](../../../guides/login/webapp-saml/)
+   .  
+    See the following table for details.
+
+    <table>
+    <thead>
+    <tr class="header">
+    <th>Field</th>
+    <th>Value</th>
+    <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="odd">
+    <td>Issuer</td>
+    <td><div class="content-wrapper">
+    <p><code>                 developer.twitter.com                </code></p>
+    </div></td>
+    <td>This is the <code>               &lt;saml:Issuer&gt;              </code> element that contains the unique identifier of the service provider.</td>
+    </tr>
+    <tr class="even">
+    <td>Assertion Consumer URL</td>
+    <td><pre><code>https://developer.twitter.com/a/&lt;ENTER_YOUR_DOMAIN&gt;/acs</code></pre>
+    <code>              </code></td>
+    <td>This is the URL to which the browser should be redirected to after the authentication is successful. This is the Assertion Consumer Service (ACS) URL of the service provider. The identity provider redirects the SAML2 response to this ACS URL. However, if the SAML2 request is signed and SAML2 request contains the ACS URL, the Identity Server will honor the ACS URL of the SAML2 request.</td>
+    </tr>
+    <tr class="odd">
+    <td>NameID Format</td>
+    <td>The default value can be used here.</td>
+    <td>This defines the name identifier formats supported by the identity provider. The service provider and identity provider usually communicate with each other regarding a specific subject. That subject should be identified through a Name-Identifier (NameID), which should be in some format so that It is easy for the other party to identify it based on the format. Name identifiers are used to provide information regarding a user.</td>
```

Comment:
```suggestion
    <td>This defines the name identifier formats supported by the identity provider. The service provider and identity provider usually communicate with each other regarding a specific subject. That subject should be identified through a Name-Identifier (NameID), which should be in some format so that it is easy for the other party to identify it based on the format. Name identifiers are used to provide information regarding a user.</td>
```